examples/website/mask-extension/package.json

```diff
@@ -14,6 +14,7 @@
     "@material-ui/icons": "^4.9.1",
     "@types/react": "^18.0.0",
     "@types/react-dom": "^18.0.0",
+    "d3-dsv": "^3.0.1",
```

Comment:
Runs fine for me without this. This module is not mentioned anywhere in the code

Reply:
I see this error when I try to run it without installing `d3-dsv`
```
examples/website/mask-extension on  master [!] 
❯ yarn start
yarn run v1.22.11
$ vite --open

  VITE v4.5.5  ready in 350 ms

  ➜  Local:   http://127.0.0.1:5173/
  ➜  Network: use --host to expose
  ➜  press h to show help
✘ [ERROR] Could not resolve "d3-dsv"

    node_modules/@loaders.gl/csv/dist/lib/encoders/encode-csv.js:6:30:
      6 │ import { csvFormatRows } from 'd3-dsv';
        ╵                               ~~~~~~~~

  You can mark the path "d3-dsv" as external to exclude it from the bundle, which will remove this
  error.

/Users/wille/projects/deck.gl/examples/website/mask-extension/node_modules/esbuild/lib/main.js:1649
  let error = new Error(`${text}${summary}`);
              ^

Error: Build failed with 1 error:
node_modules/@loaders.gl/csv/dist/lib/encoders/encode-csv.js:6:30: ERROR: Could not resolve "d3-dsv"
    at failureErrorWithLog (/Users/wille/projects/deck.gl/examples/website/mask-extension/node_modules/esbuild/lib/main.js:1649:15)
    at /Users/wille/projects/deck.gl/examples/website/mask-extension/node_modules/esbuild/lib/main.js:1058:25
    at /Users/wille/projects/deck.gl/examples/website/mask-extension/node_modules/esbuild/lib/main.js:1525:9
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  errors: [
    {
      detail: undefined,
      id: '',
      location: {
        column: 30,
        file: 'node_modules/@loaders.gl/csv/dist/lib/encoders/encode-csv.js',
        length: 8,
        line: 6,
        lineText: "import { csvFormatRows } from 'd3-dsv';",
        namespace: '',
        suggestion: ''
      },
      notes: [
        {
          location: null,
          text: 'You can mark the path "d3-dsv" as external to exclude it from the bundle, which will remove this error.'
        }
      ],
      pluginName: '',
      text: 'Could not resolve "d3-dsv"'
    }
  ],
  warnings: []
}
error Command failed with exit code 1.
```

Reply:
This appears to be a missing dependency in `@loaders.gl/csv`, could you report the [issue](https://github.com/visgl/loaders.gl/issues) there?

Reply:
FWIW: https://github.com/visgl/loaders.gl/blob/master/modules/csv/package.json#L49

Reply:
Perhaps it is an issue in an earlier version of loaders?